crypto/include/fdoCryptoHal.h

```diff
@@ -133,24 +131,30 @@ int32_t crypto_hal_aes_decrypt(uint8_t *clear_text, uint32_t *clear_text_length,
 			       const uint8_t *cypher_text,
 			       uint32_t cypher_length, size_t block_size,
 			       const uint8_t *iv, const uint8_t *key,
-			       uint32_t key_length);
+			       uint32_t key_length,
+				   uint8_t *tag, size_t tag_length,
+				   const uint8_t *aad, size_t aad_length);
 
-/* AES-GCM authenticated encryption/decryption APIs */
-int32_t fdo_crypto_aes_gcm_encrypt(const uint8_t *plain_text,
+
+
+/* AES-CCM authenticated encryption/decryption APIs */
+int32_t fdo_crypto_aes_ccm_encrypt(const uint8_t *plain_text,
 				   uint32_t plain_text_length,
 				   uint8_t *cipher_text,
 				   uint32_t cipher_text_length,
 				   const uint8_t *iv, uint32_t iv_length,
 				   const uint8_t *key, uint32_t key_length,
-				   uint8_t *tag, uint32_t tag_length);
+				   uint8_t *tag, uint32_t tag_length,
+				   const uint8_t *aad, size_t aad_length);
 
-int32_t fdo_crypto_aes_gcm_decrypt(uint8_t *clear_text,
```

Comment:
They were only defined for storage. Since now we make calls to the HAL APIs directly, we don't need them anymore.
Removed the missed out declaration for fdo_crypto_aes_ccm_{encrypt,decrypt} as well